windows-installer/setup.nsi

```diff
@@ -79,8 +82,9 @@ RequestExecutionLevel admin
 ; FUNCTIONS
 ;--------------------------------
 
-; Define StrTrimNewLines function to use below.
+; Define StrTrimNewLines and StrRep functions to use below.
```

Comment:
Ok, now I'm a bit confused... Why do we need to do this explicitly for `StrTrimNewLines` and `StrRep`, but not for, e.g., `WordFind`? Are these lines superfluous?